keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -293,33 +292,61 @@ def _augment(self, inputs):
         return result
 
     def _batch_augment(self, inputs):
-        return self._map_fn(self._augment, inputs)
+        augmented_batch = self._map_fn(self._augment, inputs)
+        return augmented_batch
 
     def _format_inputs(self, inputs):
+        metadata = {"is_dict": False, "use_targets": False}
```

Comment:
is_dict should probably default to true so we don't have to rewrite it everywhere